chat/consumers.py

```diff
@@ -0,0 +1,195 @@
+import json
+
+from channels.db import database_sync_to_async
+from channels.generic.websocket import AsyncWebsocketConsumer
+from django.contrib.auth.models import User
```

Comment:
I'd recommend using `get_user_model` in case in any moment we decide to change the user model.

```suggestion
from django.contrib.auth import get_user_model
```